File: 2292-counting-words-with-a-given-prefix/2292-counting-words-with-a-given-prefix.java
class Solution {
    public int prefixCount(String[] w, String p) {
        int n = p.length() , count = 0;
        for(int i=0;i<w.length;i++)
        {
            if(w[i].length()<n)
            {
                continue;
            }
            boolean flag=true;
            for(int j=0;j<n;j++)
            {
                if(p.charAt(j)!=w[i].charAt(j))
                {
                    flag=false;
                    break;
                }
            }
            if(flag)
            {
                count++;
            }
        }
        return count;        
    }
}

File: 2292-counting-words-with-a-given-prefix/2292-counting-words-with-a-given-prefix.py
class Solution:
    def prefixCount(self, w: List[str], p: str) -> int: # w ;: words , p : pref
        cnt=0 # cnt = count
        for i in w: # i : interation of each words
            if i.startswith(p):
                cnt+=1
        return cnt
        